languages/csharp/exercises/concept/inheritance/Inheritance.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+
+abstract class Character
+{
+    protected Character(string characterType)
+    {
+        throw new NotImplementedException("Please implement the Character() constructor");
+    }
+
+    public abstract int DamagePoints(Character target);
+
+    public virtual bool Vulnerable()
+    {
+        throw new NotImplementedException("Please implement the Character.Vulnerable() method");
+    }
+
+    public override string ToString()
+    {
+        throw new NotImplementedException("Please implement the Character.ToString() method");
+    }
+}
+
+class Warrior : Character
+{
+    public Warrior() : base("TODO")
```

Comment:
I've opted to provide this stub constructor to have the code compile. The students will have defining constructors as a prerequisite. The only new things is the `base` call, but I figured I'd just stub out the call with an invalid value.